src/couch_mrview/src/couch_mrview_http.erl

```diff
@@ -43,6 +43,11 @@
 -include_lib("couch/include/couch_db.hrl").
 -include_lib("couch_mrview/include/couch_mrview.hrl").
 
+-define(DEFAULT_ENABLE_KEY_EXCLUSIVITY, false).
```

Comment:
we do the defaults inline everywhere else and this define is used exactly once here, so I would prefer to just say `false` in the config:get_boolean line below.